test/e2e/tests/test_queue.py

```diff
@@ -137,3 +141,26 @@ def test_crud(self, simple_queue):
         tags.assert_equal_without_ack_tags(
             expect_after_update_tags, latest_tags,
         )
+
+
+class TestAdoptQueue(adoption.AbstractAdoptionTest):
```

Comment:
Yay! I'm glad someone is able to get value out of this adoption test abstraction!